### ISSUE TITLE
display_ipython not returning visuals.

### ISSUE THREAD

**Issue by extraymond** [pydy/pydy#388]
**Environments**

* ubuntu 16.04 x64
* python 3.6 installed from miniconda
* pydy 0.4 from conda(conda-forge)
* Tested under both chromium and firefox.

**Expected behaviour**

Embedding 3D window below the self, and return visuals like display().

**Encountered behaviour**

1. No objects found in the executed cell
2. No 3D window embedded below.
3. Variable's and initial condition setting window is working.

**Screenshot**
1. Results with display_ipython()
![results with display_ipython()](https://user-images.githubusercontent.com/707176/34999519-d6200c56-fb1c-11e7-82ba-bd80127c4586.png)
2. Results with display()
![results with display()](https://user-images.githubusercontent.com/707176/34999521-d816cf40-fb1c-11e7-97ce-fbbd53de3610.png)

**Comment by oliverlee** [comment]
You can modify the cell like this until a fix has been merged:
```python
scene.display_ipython()
import IPython.display
IPython.display.display(IPython.display.HTML(scene._html_widget.value))
```

**Comment by extraymond** [comment]
Thx!!! It's working now. I will stick to this fix for now.

**Comment by oliverlee** [comment]
`widget.HTML` (the class of `scene._html_widget`) is different from `display.HTML`. The call to `IPython.display.display` needs to create a display object and I guess it doesn't handle the `widget.HTML` object correctly (or in the way we intend).

I haven't spent too much time looking into why.

**Comment by goatchurchprime** [comment]
May as well include this two-line work-around in the code of the notebook examples, eg chaos_pendulum

These animations are an important route in for noobs who don't yet know why eigenvalues of the linearized equations are interesting, so it could be important to make work.

**Comment by moorepants** [comment]
I recommend using `display_jupyter()` now.

**Comment by moorepants** [comment]
Actually, `display_jupyter()` is only tin the development version. I'll try to make a new release soon.

**Comment by Simone2021space** [comment]
Hi,
is it possible to visualize 3D space?

**Comment by moorepants** [comment]
Yes. The examples in the dev docs show this: https://pydy.readthedocs.io/en/latest/examples/multidof-holonomic.html

**Comment by Simone2021space** [comment]
Thanks,
and it is possible to draft parallelepiped, in addition to cubes?
I don't think they are on the available shapes

**Comment by moorepants** [comment]
Everything in shapes is supported: https://pydy.readthedocs.io/en/latest/viz/pymodules/shapes.html
